lib/client-assets.php

```diff
@@ -167,6 +167,11 @@ function gutenberg_register_scripts_and_styles() {
 		),
 		'after'
 	);
+	wp_add_inline_script(
+		'wp-api-fetch',
```

Comment:
I think this addition makes more sense as an addition to the `heartbeat` script instead of `wp-api-fetch`

Reply:
You would have also to add `wp-hooks` as a dependency of `wp-api-fetch` because it is now used in the code.

It might be more tricky to put it as part of `heartbeat` because of `wp-hooks` dependency.

Reply:
`wp-hooks` is now as a dependency of `wp-api-fetch`

Reply:
I still think ideally, this script should be attached to heartbeat. It's possible to do so https://wordpress.stackexchange.com/questions/100709/add-a-script-as-a-dependency-to-a-registered-script 

This allows us to remove the `jQuery` dependency from the `wp-api-fetch` script handle as well. Ideally, we would port `heartbeat` to a gutenberg package and just do it in the code instead of an inline script. 

Anyway, I'm fine with this change if we add a comment explaining what we should do ideally.

Reply:
Thanks @youknowriad for the suggestion! 👍 

I have moved the addition to the heartbeat script. Does it look good now? 

I'm not very familiar yet with PHP or the WordPress API, so it may be possible I forgot something else. Let me know if that it's the case!

What I have not found is where the `jQuery` dependency is defined for the `wp-api-fetch` script. Where is it? It looks like only the `wp-edit-post` script is defining that dependency.